.github/workflows/functional-tests-workflow.yml

```diff
@@ -35,10 +35,10 @@ jobs:
         with:
           ref: ${{ inputs.checkout-ref }}
           repository: ${{ inputs.checkout-repository }}
-      - name: Set up Python 3.10
+      - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.10'
+          python-version: '3.8'
```

Comment:
Why downgrading to 3.8 ?

Reply:
Originally I put 3.11 because it was slightly faster, but @svdimchenko suggested the oldest version to be sure that all works as expected. Generally older versions are the most problematic, as we tend to use the newest python capabilities.